prep-html-links/index.html

```diff
@@ -7,6 +7,6 @@
 </head>
 <body>
   <h1>Home page</h1>
-<a href= "file:///Users/dadsmac/repos/prep-0621-code-solutions/prep-html-links/contact-us/contact-us.html">Click here to go to the contact page</a>;
+<a href= "file:///Users/dadsmac/repos/prep-0621-code-solutions/prep-html-links/contact-us/contact-us.html">Click here to go to the contact page</a>
```

Comment:
Ok, one down, one to go! This link should also be "relative", using just `"contact-us/contact-us.html"` instead of the full file path.

Reply:
https://user-images.githubusercontent.com/85319436/121221972-f6d8e500-c83a-11eb-8a47-446cc5843498.mov

Reply:
This works when you click the links, but we want to avoid forcing whoever downloads your code to have an identical file system to your computer. They don't have `/Users/dadsmac` on their machine, so if you cut it down to just `contact-us/contact-us.html`, it'll work when you click the link, but also be system-independent.